tslint.js

```diff
@@ -7,18 +7,19 @@ module.exports = {
     'tslint-eslint-rules',
     'tslint-microsoft-contrib',
     'tslint-config-prettier',
-    'tslint-react-hooks'
+    'tslint-react-hooks',
```

Comment:
Extra comma?

Reply:
this package had prettier's `trailingComma` setting turned on. Not sure why it didn't fix this file last time it was updated though.